scripts/bug_classifier.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+
+import argparse
+import os
+
+import numpy as np
+
+from bugbug import bugzilla
+from bugbug.models import get_model_class
+
+
+def classify_bugs(model_name, classifier):
+    classifier_models = ["component"]
```

Comment:
My idea here is that this will make it easier to add more models that have a classifier argument. I can change it to what was originally in run.py if you prefer.

Reply:
No, that's a good idea. My only suggestion:
1) If classifier != "default", assert model_name is in this list, to make sure the parameters passed by the user are valid
2) Rename the variable, as `classifier_models` is too generic (maybe `MODELS_WITH_TYPE`, and make it a constant (UPPERCASE and at the top of the file after the imports) tuple.